sros2/sros2/verb/generate_policy.py

```diff
@@ -126,24 +68,68 @@ def main(self, *, args):
 
         with DirectNode(args) as node:
             for node_name in node_names:
-                profile = self.get_profile(policy, node_name)
+                profile = policy.get_or_create_profile(node_name.node, node_name.ns)
+
                 subscribe_topics = get_subscriber_info(node=node, node_name=node_name)
-                if subscribe_topics:
-                    self.add_permission(
-                        profile, 'topic', 'subscribe', 'ALLOW', subscribe_topics, node_name)
+                for topic in subscribe_topics:
+                    _add_subscribe_topic_expression(profile, node_name, topic)
```

Comment:
I guess `ALLOW` is made implicit here?
Should we make it a default argument to the function signature?
Or never mind as these private function are outside the larger /profile/ api?

Reply:
Indeed, this is just breaking up logical steps into functions. It's private and pretty specific, so I think it's okay to assume `ALLOW`. If in the future we need to make it more generically useful, that would be super easy.